static/js/components/VideoCard.js

```diff
@@ -20,8 +22,10 @@ const VideoCard = (props: VideoCardProps) => (
         <a href={makeVideoUrl(props.video.key)}>{props.video.title}</a>
       </h4>
       <div className="actions">
-        <i className="material-icons">share</i>
-        <i className="material-icons">mode_edit</i>
+        {
+          props.isAdmin &&
+          <a className="material-icons" onClick={props.showDialog}>mode_edit</a>
```

Comment:
Position of edit button is affected by length of video title; maybe force it to the bottom right via CSS:

![screen shot 2017-09-12 at 12 48 59 pm](https://user-images.githubusercontent.com/187676/30338270-15d04fae-97b9-11e7-891e-7ffcfca24bbd.png)

Reply:
OK, will fix

Reply:
Still seeing the button at varying heights above the bottom of the card depending on the number of lines taken up by the title, but it doesn't seem like a big issue.

Reply:
Here's a recent screenshot:

![ss 2017-09-12 at 15 20 32](https://user-images.githubusercontent.com/14932219/30344124-07c4cb6e-97ce-11e7-9301-947a444d368f.png)

Do you have a different expectation? Maybe @roberthouse54 should chime in?

Reply:
I was thinking that the edit button should always appear at the far bottom right corner of the video card all the time (about where it is in the first card with the long title), instead of at different heights above the bottom of the card, but it’s just a minor nitpick.